src/main/kotlin/com/criptext/mail/scenes/mailbox/MailboxSceneController.kt

```diff
@@ -151,6 +152,7 @@ class MailboxSceneController(private val scene: MailboxScene,
         override fun onToggleThreadSelection(thread: EmailPreview, position: Int) {
             if (!model.isInMultiSelect) {
                 changeMode(multiSelectON = true, silent = false)
+                observer.onStartGuideMultiple()
```

Comment:
Should not use observer here, you should call the scene to show the select multiple guide and use the observer to let know the controller whatever the scene need it to know after or during showing of that guide.